auto.js

```diff
@@ -791,10 +805,20 @@ var CookieAuto = {};
 
                     let wthresh = q('#buyscript_wthresh')[0]
                     wthresh.oninput = wthresh.onchange = () => {
-                        q('#buyscript_wthreshRightText')[0].innerHTML = wthresh.value + '%';
-                        settings.wrinklerThreshold = wthresh.value / 100;
+                        settings.wrinklerThreshold = Math.round(Math.pow(3600, wthresh.value / 1000) * 24);
+                        q('#buyscript_wthreshRightText')[0].innerHTML = TimeBeautify(settings.wrinklerThreshold * 1000);
                         saveSettings();
                     }
+
+                    let ascendRatioInput = q('#buyscript_arat')[0]
+                    ascendRatioInput.oninput = ascendRatioInput.onchange = () => {
```

Comment:
I should mention... If both events are called in the event of an input to an element, the callback is called twice, just so that we are aware.